client/jetpack-connect/plans-extended-info.jsx

```diff
@@ -0,0 +1,55 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormattedHeader from 'components/formatted-header';
+
+const PlansExtendedInfo = ( { onClick, translate } ) => {
+	const headerText = translate( 'Unsure which plan is right for you?' );
+	const subheaderText = translate(
+		'Read out guide to choosing the right plan or compare plans in more details.'
+	);
+
+	return (
+		<Card className="jetpack-connect__plan-info">
+			<img
+				className="jetpack-connect__plan-info-illustration"
+				src={ '/calypso/images/illustrations/illustration-start-jetpack.svg' }
+				alt=""
+			/>
+			<FormattedHeader
+				className="jetpack-connect__plan-help"
+				headerText={ headerText }
+				subHeaderText={ subheaderText }
+			/>
+			<div className="jetpack-connect__plan-info-buttons">
+				<Button
+					primary
+					href={
+						'https://jetpack.com/2017/02/01/a-simple-guide-to-choosing-the-best-jetpack-plan-your-wordpress-site/'
+					}
+					onClick={ onClick( 'plan_guide' ) }
+				>
+					{ translate( 'PLAN GUIDE' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Plan Guide' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Could we use the string we have already, but use CSS's `text-transform: uppercase` instead?

Reply:
yes, I will add all the suggestions after the patch is reviewed.